potassco/clingo.h

```diff
@@ -71,6 +71,8 @@ class AbstractAssignment {
 	virtual bool     hasConflict()     const = 0;
 	//! Returns the number of decision literals in the assignment.
 	virtual uint32_t level()           const = 0;
+	//! Returns the the number of decision literals that will not be backtracked while solving.
```

Comment:
I think it's good to have this functionality but I don't like the name of the function. Given that we already have isFiexd(), I'd suggest something like fixed().